services/workflows-service/src/app.module.ts

```diff
@@ -124,6 +125,7 @@ export const validate = async (config: Record<string, unknown>) => {
     initHttpMoudle(),
     RuleEngineModule,
     NotionModule,
+    StatisticsModule,
```

Comment:
Would be confusing to have both a metrics module and a statistics module. Just a naming thing.